Add more steps to the play with grafana mimir page (#7859)
grafana/mimir: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md

> F---

Reviewer: This looks like a typo
```suggestion
---
```

1. Open [Grafana Alerting](http://localhost:9000/alerting/list).
2. Click "New alert rule", which also allows you to configure recording rules.
3. Configure the recording rule:
   1. Select **Mimir or Loki recording rule** in the top selector.
   1. Choose **Mimir** in the **Select data source** field.
   1. Type `sum:up` in the **Rule name** field.
   1. Choose **Code** in the **Builder | Code** field on the right.
   1. Type `sum(up)` in the **Metrics browser** query field.

>    1. Choose  **Data source-managed** in the **Grafana-managed | Data source-managed** field in the **Rule Type** paragraph.

Reviewer 2: ```suggestion
   1. Choose **Data source-managed** in the **Grafana-managed | Data source-managed** field in the **Rule Type** paragraph.
```

3. Configure the recording rule:

>    1. Select `Mimir or Loki recording rule` in the top selector.

Reviewer 3: ```suggestion
   1. Select **Mimir or Loki recording rule** in the top selector.
```

Author: I was just doing copy-pasta, will apply your fixes

Reviewer 3: Yeah, I completely understand :) thanks for considering the changes!

Apologies for the inconsistent style.

As an interim owner of the docs, I'm looking to clean bits up as I go and I appreciate any contributions that can help me do that.

>    2. Choose `Mimir` in the "Select data source" field.

Reviewer 3: ```suggestion
   1. Choose **Mimir** in the **Select data source** field.
```

>    3. Type `sum:up` in the "Rule name" field.

Reviewer 3: ```suggestion
   1. Type `sum:up` in the **Rule name** field.
```

>    4. Choose "Code" in the "Builder | Code" field on the right.

Reviewer 3: ```suggestion
   1. Choose **Code** in the **Builder | Code** field on the right.
```

>    5. Type `sum(up)` in the "Metrics browser" query field.

Reviewer 3: ```suggestion
   1. Type `sum(up)` in the **Metrics browser** query field.
```

>    6. Choose  "Data source-managed" in the "Grafana-managed | Data source-managed" in the "Rule Type" paragraph.

Reviewer 3: ```suggestion
   1. Choose  **Data source-managed** in the **Grafana-managed | Data source-managed** field in the **Rule Type** paragraph.
```

>    7. Type `example-namespace` in the "Namespace" field.

Reviewer 3: ```suggestion
   1. Type `example-namespace` in the **Namespace** field.
```

>    8. Type `example-group` in the "Group" field.

Reviewer 3: ```suggestion
   1. Type `example-group` in the **Group** field.
```

>    9. From the upper-right corner, click the **Save and exit** button.

Reviewer 3: ```suggestion
   1. From the upper-right corner, click the **Save and exit** button.
```
